.dockerignore

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
They contain builds and runs as expect, but likely could have a few things excluded. Eg: .git

Happy for you to suggest or I will look properly in a day or 2.